README.md

```diff
@@ -151,6 +151,6 @@ This data is collected from the `package.json` file in the sample application an
 
 [deploy_track_url]: https://github.com/cloudant-labs/deployment-tracker
 [cloud_foundry]: https://github.com/cloudfoundry/cli
-[getting_started]: https://www.ibm.com/watson/developercloud/doc/common/index.html
-[nlc_docs]: https://www.ibm.com/watson/developercloud/nl-classifier.html
-[sign_up]: https://console.ng.bluemix.net/registration/
+[getting_started]: https://console.bluemix.net/docs/services/watson/index.html#about
+[nlc_docs]: https://www.ibm.com/watson/services/natural-language-classifier/
```

Comment:
The nlc docs are at `https://console.bluemix.net/docs/services/natural-language-classifier/getting-started.html`. This link goes to the marketing page.